public/preload/view.js

```diff
@@ -3,43 +3,59 @@
   remote,
   webFrame,
 } = require('electron');
-
-const {
-  enable: enableDarkMode,
-  disable: disableDarkMode,
-} = require('darkreader');
+const fs = require('fs');
+const path = require('path');
 
 const ContextMenuBuilder = require('../libs/context-menu-builder');
 
 const { MenuItem, shell } = remote;
 
+const loadDarkReader = () => {
+  const shouldUseDarkColor = ipcRenderer.sendSync('get-should-use-dark-colors');
+  const darkReader = ipcRenderer.sendSync('get-preference', 'darkReader');
+  let loadScript = '';
+  if (shouldUseDarkColor && darkReader) {
+    const {
+      darkReaderBrightness,
+      darkReaderContrast,
+      darkReaderGrayscale,
+      darkReaderSepia,
+    } = ipcRenderer.sendSync('get-preferences');
+    console.log(`DarkReader.setFetchMethod(window.fetch);
```

Comment:
Seems like you could just do this after the following and `console.log(loadScript)`

Reply:
Thank you, Stefan! I actually rewrote it here: https://github.com/atomery/juli/pull/92/files

Reply:
OK, so how do I get it to update the Juli version to 9.6.0? 
<img width="595" alt="Screen Shot 2020-08-30 at 2 05 45 PM" src="https://user-images.githubusercontent.com/157270/91669446-ef292a80-eac9-11ea-8e34-534d6411cf12.png">

Reply:
It’s up now :)